Dockerfile

```diff
@@ -1,14 +1,17 @@
 # This is a template to be used by GoReleaser.
 # See docs for details: https://goreleaser.com/customization/docker/
 
-FROM alpine:3.16
-WORKDIR /app
+FROM golang:alpine
+
+RUN wget -O /usr/bin/yt-dlp https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp
+RUN chmod +x /usr/bin/yt-dlp
+RUN ln -s /usr/bin/yt-dlp /usr/bin/youtube-dl
+RUN apk --no-cache add ca-certificates python3 py3-pip ffmpeg tzdata
 
-RUN wget -O /usr/bin/yt-dlp https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp && \
-    chmod +x /usr/bin/yt-dlp && \
-    ln -s /usr/bin/yt-dlp /usr/bin/youtube-dl && \
-    apk --no-cache add ca-certificates python3 py3-pip ffmpeg tzdata
-COPY podsync /app/podsync
+COPY . /tmp/podsync
 
-ENTRYPOINT ["/app/podsync"]
-CMD ["--no-banner"]
+WORKDIR /tmp/podsync
+RUN go build -o /app/podsync ./cmd/podsync
```

Comment:
This will break goreleaser. See header comment above.

Reply:
could you please suggest on how to test goreleaser?

Reply:
I'd just add another file (like `Dockerfile.build`) and keep these side by side.

Reply:
Ppl will publish gist first and then we'll see the warning :)
Possibly we could offer an example how to download gist via script and save it to a disk before running podsync?

Something like
```
curl gist > ~/config.toml
podsync --config ~/config.toml
```